schemas/triniti/ovp/event/transcoding-completed/latest.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<pbj-schema xmlns="http://gdbots.io/pbj/xsd"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://gdbots.io/pbj/xsd http://gdbots.io/pbj/xsd/schema.xsd">
+
+  <schema id="pbj:triniti:ovp:event:transcoding-completed:1-0-0">
+    <fields>
+      <field name="node_ref" type="identifier" required="true">
+        <php-options>
+          <imports>use Gdbots\Schemas\Ncr\NodeRef;</imports>
+          <class-name>NodeRef::class</class-name>
+        </php-options>
+        <js-options>
+          <imports>import NodeRef from '@gdbots/schemas/gdbots/ncr/NodeRef';</imports>
+          <class-proto>NodeRef</class-proto>
+        </js-options>
+      </field>
+      <field name="job_identifier" type="string" pattern="^[\w:-]+$"/>
```

Comment:
@efjacobson is the node_ref here the video asset ref or the video ref?  we should add a description to clarify it.  also is the job_identifier meant to be generic or is that aws vendor specific?  we should probably make that explicit.  do we need a job arn or just the id?

Reply:
@gdbrown 

- for ref, thats a good question. i thought it might be interesting to leave that open ended but maybe its dangerously confusing. i think we're leaning towards keeping track of this on assets so i can add a description for that
- for the identifier, you mentioned something about describing the workflow generically which is why i chose `job_identifier`. but if you want it aws specific i can make it arn

Reply:
@efjacobson workflow generic (command/events) but the fields, I'd store the whole arn, like mediaconvert_job_arn -- similar to medialive stuff.

Reply:
@gdbrown they dont have arns but i think queue + id is as close as you can get